src/components/creator/Editor/SceneEdition/SceneEdition.tsx

```diff
@@ -33,16 +34,17 @@ export const relocateActor = (row: string[], colSearch: number, inMap: SceneMap)
 }
 
 type SizeProps = {
-    mapIndex: number
     setColumns: (col: number) => void
     setRows: (row: number) => void
 }
 
 const SizeEditor = (props: SizeProps) => {
     const { t } = useTranslation("creator")
 
-    const initialRows = LocalStorage.getCreatorChallenge()?.scene.maps[props.mapIndex].length
-    const initialColumns = LocalStorage.getCreatorChallenge()?.scene.maps[props.mapIndex][INITIAL_ROW].length
+    const { currentMap } = useContext(CreatorContext)
+
+    const initialRows = LocalStorage.getCreatorChallenge()?.scene.maps[currentMap.index].length
+    const initialColumns = LocalStorage.getCreatorChallenge()?.scene.maps[currentMap.index][INITIAL_ROW].length
```

Comment:
¿Esto no debería tomarse del context?

```suggestion
    const initialRows = currentMap.map.length
    const initialColumns = currentMap.map[INITIAL_ROW].length
```